src/ui/public/index_patterns/_field_format/contentTypes.js

```diff
@@ -6,6 +6,10 @@ export default function contentTypesProvider(highlightFilter) {
   var types = {
     html: function (format, convert) {
       return function recurse(value, field, hit) {
+        if (typeof value === 'undefined') {
```

Comment:
Also, the `value == null` bit here works when value is `undefined` or `null`, which I think is a desirable addition.